aspnetcore/log-mon/metrics/metrics.md

```diff
@@ -20,7 +20,8 @@ Metrics are numerical measurements reported over time. They're typically used to
 
 These metrics can be reported to a monitoring system at regular intervals. Dashboards can be setup to view metrics and alerts created to notify people of problems. If the web service is intended to respond to requests within 400 ms and starts responding in 600 ms, the monitoring system can notify the operations staff that the app response is slower than normal.
 
-See [ASP.NET Core metrics](https://github.com/dotnet/aspnetcore/issues/47536) for ASP.NET Core specific metrics. See [.NET metrics](/dotnet/core/diagnostics/metrics) for .NET metrics.
+> [!TIP]
```

Comment:
Not a fan of `TIP` bling but won't protest too much.

<img width="364" alt="image" src="https://github.com/dotnet/AspNetCore.Docs/assets/3605364/a8ae84f2-d8ff-4abb-835a-ae4e0adc08bf">